cmds/bmp/bms_command_test.go

```diff
@@ -70,13 +75,69 @@ var _ = Describe("bms command", func() {
 			Expect(validate).To(BeTrue())
 			Expect(err).ToNot(HaveOccurred())
 		})
+
+		Context("bad BmsCommand", func() {
+			Context("no deployment file", func() {
+				BeforeEach(func() {
+					options = cmds.Options{
+						Verbose: false,
+						Deployment: "",
+					}
+				})
+
+				It("fails validation", func(){
+					cmd = bmp.NewBmsCommand(options, fakeBmpClient)
+					validate, err := cmd.Validate()
+					Expect(validate).To(BeFalse())
+					Expect(err).To(HaveOccurred())
+				})
+			})
+
+			Context("deployment file doesn't exist", func() {
+				BeforeEach(func() {
+					options = cmds.Options{
+						Verbose: false,
+						Deployment: "fake-deployment-file",
+					}
+				})
+
+				It("fails validation", func(){
+					cmd = bmp.NewBmsCommand(options, fakeBmpClient)
+					validate, err := cmd.Validate()
+					Expect(validate).To(BeFalse())
+					Expect(err).To(HaveOccurred())
+				})
+			})
+		})
 	})
 
 	Describe("#Execute", func() {
-		It("executes a good BmsCommand", func() {
-			rc, err := cmd.Execute(args)
-			Expect(rc).To(Equal(0))
-			Expect(err).ToNot(HaveOccurred())
+		Context("executes a good BmsCommand", func() {
+			BeforeEach(func() {
+				fakeBmpClient.BmsResponse.Status = 200
+				fakeBmpClient.BmsErr = nil
+			})
+
+			It("executes with no error", func() {
+				rc, err := cmd.Execute(args)
```

Comment:
I would rather see this be more explicit, e.g., `cmd.Execute([]string{"bms", "bms"})` or similar.